src/ambianic/__init__.py

```diff
@@ -27,7 +27,12 @@ def __merge_secrets(config:Union[Dynaconf, DynaBox], src_config:Dynaconf = None)
         if isinstance(val, dict):
             __merge_secrets(val, src_config)
             continue
-        if isinstance(val, str) and val[0:2] == "${":
```

Comment:
@muka I am not sure why you are parsing for ${} expressions during config load. [DynaConf seems to have](https://dynaconf.readthedocs.io/en/docs_223/guides/environment_variables.html#merging-new-data-to-existing-variables) a flexible way to do that for environment and runtime variables.

If you are thinking about populating a notification message from its template string, then that should probably happen right before the message is sent. And for that purpose, [string.format() seems appropriate](https://www.w3schools.com/python/ref_string_format.asp):

```
notification_message=notification_template.format(**template_vars))
```

Reply:
This is specifically crafted to apply secrets replacement.

Reply:
I see. Where do the secrets values come from? Not environment? Is there a doc reference how users can use this?